docs/docs/launch-manual/sections/core-hacking/sections/publishing.md

```diff
@@ -0,0 +1,93 @@
+## Publishing
+
+Pulsar bundles a command line utility called `ppm` which we first used back in
+[Command Line](../../using-pulsar/#command-line) to search for and install
+packages via the command line. This is invoked by using the `pulsar` command
+with the `-p` or `--package` option. The `pulsar -p` command can also be used to
+publish Pulsar packages to the public registry and update them.
+
+See more in [Using PPM](#using-ppm).
+
+### Prepare Your Package
+
+There are a few things you should double check before publishing:
+
+- Your `package.json` file has `name`, `description`, and `repository` fields.
+- Your `package.json` file has a `version` field with a value of `"0.0.0"`.
+- Your `package.json` file has an `engines` field that contains an entry for
+  `atom` such as: `"engines": {"atom": ">=1.0.0 <2.0.0"}`.
+- Your package has a `README.md` file at the root.
+- Your `repository` URL in the `package.json` file is the same as the URL of
+  your repository.
+- Your package is in a Git repository that has been pushed to
+  [GitHub](https://github.com). Follow [this guide](https://help.github.com/articles/importing-a-git-repository-using-the-command-line/)
+  if your package isn't already on GitHub.
+
+### Publish Your Package
+
+Before you publish a package it is a good idea to check ahead of time if a
+package with the same name has already been published to
+[the Pulsar Package Repository](https://web.pulsar-edit.dev/packages). You can
+do that by visiting `https://web.pulsar-edit.dev/packages/your-package-name` to
+see if the package already exists. If it does, update your package's name to
+something that is available before proceeding.
+
+Now let's review what the `pulsar -p publish` command does:
+
+1. Registers the package name on Pulsar Package Repository if it is being
+   published for the first time.
+2. Updates the `version` field in the `package.json` file and commits it.
+3. Creates a new [Git tag](https://git-scm.com/book/en/Git-Basics-Tagging) for
+   the version being published.
+4. Pushes the tag and current branch up to GitHub.
+5. Updates Pulsar Package Repository with the new version being published.
+
+Now run the following commands to publish your package:
+
+```sh
+$ cd path-to-your-package
+$ pulsar -p publish minor
+```
+
```

Comment:
```suggestion
<!-- TODO: Rewrite this Section once Authentication Information is Public -->
```

Really maybe it's a good idea to have authentication in it's own little doc and just link to it from here? Since as of now there will be a few ways to authenticate

Reply:
Yeah makes sense to me, this is really just to get the framework in so if we get detailed docs for this then absolutely it should be split into its own section.